src/connectors/utils.py

```diff
@@ -139,3 +140,8 @@ def load_sqlfmt(file, params):
     # gen vars
```

Comment:
Is this the doc string for the `appy_part` method? If so, it might be better to place the doc string inside the function.

Reply:
I think it's a stray comment I added awhile ago to pack.py that made its way into utils

Reply:
I see. Might be okay to remove it. But also, maybe add a docstring as I wasn't able to fully grok what the `apply_part` is doing at first glance.